recipes-support/salt/salt_3000.2.bb

```diff
@@ -107,6 +108,10 @@ do_install_append() {
     cp -r ${S}/tests/ ${D}${PYTHON_SITEPACKAGES_DIR}/${PN}-tests/
 }
 
+do_install_ptest_append() {
+    install -m 0755 ${WORKDIR}/run-ptest ${D}${PTEST_PATH}
```

Comment:
This is already [a standard part](https://github.com/ni/openembedded-core/blob/nilrt/master/hardknott/meta/classes/ptest.bbclass#L44) of the `ptest.bbclass:do_install_base()`. So you can remove it; I think. :)

Sorry; I should have noticed that on my first review.